app/Resources/views/index.html.twig

```diff
@@ -9,8 +9,9 @@
 
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    <title>mozilla.ch</title>
-    <meta name="description" content="">
+    <title>Mozilla Switzerland</title>
+    <meta name="description" content="Mozilla Switzerland is part of the global community of Mozilla Foundation, founded to manifest the state of Mozilla community in Switzerland. Internet By The People Internet ...">
```

Comment:
The last part " Internet By The People Internet ..." seems a bit strange. It doesn't really make sense, plus the three dots have a space in front of them.